docker/bookkeeper/entrypoint.sh

```diff
@@ -74,6 +77,24 @@ create_bookie_dirs() {
   done
 }
 
+# Create a Bookie ID if this is a newly added bookkeeper pod
+# or read the Bookie ID if it already exists
+set_bookieid() {
+  if [ `find ${BK_ID_FILE} | wc -l` -gt 0 ]; then
+    echo "Found Bookie ID"
+    BK_bookieId=`cat ${BK_ID_FILE}`
+    export BK_bookieId
+  else
+    echo "Bookie ID not found"
+    TIMESTAMP="`date "+%Y%m%d-%H%M%S"`"
+    BK_bookieId="${HOST}-${TIMESTAMP}"
```

Comment:
What about using some random UUID instead of a timestamp?
This ID will stay there forever for the whole life of the bookie and you will see it in every log file.
The timestamp may be confusing for people who read the logs

Reply:
Have incorporated this change